### PR TITLE
Navbar hamburger and toggle state in small screens

### PR DESCRIPTION
**Pull request by bharati-21**
Added navbar toggle state toggle functionality in small screens
- Added a close icon to close the opened tray
- Added class `navbar-open` that indicates that the navbar is in open state.

### CODE REVIEW

**Review by poornima-alugubelly — APPROVED** [review]
LGTM!

**Comment by bharati-21** [comment]
> LGTM!

Thank you!

**Review by manojasarsa — APPROVED** [review]
Code is clean and well structured. Good to merge!

**Comment by bharati-21** [comment]
> Code is clean and well structured. Good to merge!

Thank you!

File: pages/Authentication/logout/index.html [hunk 1]
```diff
@@ -217,5 +222,7 @@ <h3 class="text-uppercase auth-head flex-row flex-align-center flex-justify-cent
                 </div>
             </div>           
         </footer>
+        <script src="/scripts/hamburger.js"></script>
+        <script src="/scripts/hamburger.js"></script>
```
Comment (MaruthiPothuganti) [inline pages/Authentication/logout/index.html:226]: Typo

**Review by MaruthiPothuganti — COMMENTED** [review]
The code is clean and easily readable. Made good use of utility classes.

File: pages/Authentication/logout/index.html (hunk 1 above)
Comment (bharati-21) [reply pages/Authentication/logout/index.html:226]: Thank you for pointing this out! I shall update this after the review is done. 👍🏽

**Pull request merged**